habitat_baselines/common/utils.py

```diff
@@ -228,6 +228,42 @@ def generate_video(
         )
 
 
+def tensor_to_depth_images(tensor: Union[torch.Tensor, List]) -> np.ndarray:
+    r"""Converts tensor (or list) of n image tensors to list of n images.
+    Args:
+        tensor: tensor containing n image tensors
+    Returns:
+        list of images
+    """
+    images = []
+
+    for img_tensor in tensor:
+        image = img_tensor.permute(1, 2, 0).cpu().numpy() * 255
+        images.append(image)
+
+    return images
+
+
+def tensor_to_rgb_images(tensor: torch.Tensor) -> List[np.ndarray]:
```

Comment:
```suggestion
def tensor_to_brg_images(tensor: torch.Tensor) -> List[np.ndarray]:
```
We keep images in RGB format, OpenCV is in BGR.  Looks like this is returning things in BGR.